DXMainClient/DXMainClient.csproj

```diff
@@ -83,7 +83,7 @@
     <DefineConstants>TRACE;DEBUG;XNA</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
I don't know if there is/was a specific reason for targeting 4.0, but it was necessary to update to at least 4.5 for Autofac compatibility.

Reply:
Well, bad news but we actually have to use v4.0 because of WinXP support: https://docs.microsoft.com/en-us/dotnet/framework/install/on-windows-xp

Reply:
ah, that makes sense, I'll see what I can do...